ledger-tool/src/main.rs

```diff
@@ -847,6 +848,16 @@ fn main() {
                 .hidden(hidden_unless_forced())
                 .help(BlockVerificationMethod::cli_message()),
         )
+        .arg(
+            Arg::with_name("unified_scheduler_handler_threads")
+                .long("unified-scheduler-handler-threads")
```

Comment:
outside `unified-scheduler-pool` code, i think `handler thread count` is too verbose and centers around impl details. so, I used `handler threads` intentionally. on the other hand, `handler` is left for technical rigidness, because there's additional thread `scScheduler`, which are created for each scheduler. and the thread doesn't count towards to `unified-scheduler-handler-threads`.